traitsui/toolkit.py

```diff
@@ -110,6 +112,8 @@ def toolkit ( *toolkits ):
         return _toolkit
     else:
         if len( toolkits ) == 0:
+            warnings.warn(DeprecationWarning(
```

Comment:
A simple test using https://docs.python.org/2/library/warnings.html#available-context-managers should be possible here.